tests/bootstrap.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
We don't need this file just to output the custom error. It's perfectly fine for the process to fail without the message, the error itself is pretty self explanatory.

Reply:
fixed in 3c414ab583369f0efd84cedbf81545c4c0912a14